### ISSUE TITLE
Validate the DID

### ISSUE THREAD

**Issue by ghost** [Gimly-Blockchain/eosio-did-resolver#1]
Complete the function

```js
function checkDID(parsed: ParsedDID) {
    // TODO check DID is valid format and structure
}
```

https://github.com/Gimly-Blockchain/eosio-did-resolver/blob/master/src/resolver.ts#L3

**Comment by Balurat** [comment]
#7 
Just created a pull request for this issue, but I couldn't link it there :/

**Comment by ghost** [comment]
Resolved in PR #8
